src/openscm_runner/adapters/ciceroscm_adapter/ciceroscm.py

```diff
@@ -55,8 +56,12 @@ def get_version(cls):
         OSError
             The CICERO-SCM binary cannot be run on the operating system
         """
+        if platform.system() == "Windows":
```

Comment:
Can we use `get_executable` for this rather than duplicating the code?

Reply:
Done